scripts/bioconda-import/bioconda_importer.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python
+
+import json
+import os
+import yaml
+
+import pickle
+
+#from ruamel.yaml import YAML
+from pathlib import Path
+
+import jinja2
+
+from jinja2 import Environment
+from collections import defaultdict
+
+def fake(foo, **args):
+    pass
+
+def parse_biotools(directory):
+    """
+    Function to get biotools content data into memory.
+    """
+    data = dict()
+    for root, dirs, files in os.walk(directory):
+        for filename in files:
+            if not filename.endswith("oeb.json") and filename.endswith(".json"):
+                filepath = os.path.join(root, filename)
+                with open(filepath, 'r') as f:
+                    biotools = json.load(f)
+                    bio_id = biotools['biotoolsID'].lower()
+                    data[bio_id] = {'data': biotools, 'path': filepath}
+    return data
+
+def parse_bioconda(directory):
+    """
+    Function to get bioconda content data into memory.
+    """
+
+    data = dict()
+    for p in Path(directory).glob('./*/meta.yaml'):

```

Comment:
not sure if this will be enough to get all the recipes - some of the bioconda recipes have additional subdirectories, e.g. https://github.com/bioconda/bioconda-recipes/tree/master/recipes/gromacs

Reply:
subdirs are hopefully always old versions. I guess we make our life simpler if you try to collect only the latest version and assume that metadata is most recent in this version.

Reply:
This is not the case e.g. for gromacs, but yes, I agree it's better to keep things simple

Reply:
let's fix gromacs :)

Reply:
does this mean this can get merged?

Reply:
@hmenager 
wait, I will create a GHA job for that